projects/core/src/occ/adapters/user/occ-user.adapter.ts

```diff
@@ -17,6 +17,10 @@ import {
   InterceptorUtil,
   USE_CLIENT_TOKEN,
 } from '../../utils/interceptor-util';
+/**
+ * @deprecated since 3.2, use `OccUserDetailAdapter` from the `@spartacus/user/detail`
+ * package or `OccUserProfileAdapter` from  the `@spartacus/user/detail` package.
```

Comment:
Packages to be renamed here to Profile or Account?

Reply:
To both actually, thanks!